salt/journal/config/srv-journal-containers.env

```diff
@@ -0,0 +1,9 @@
+{%- if pillar.elife.env == 'dev' -%}
+{%- set app_env = 'ci' -%}
+{%- else -%}
+{%- set app_env = pillar.elife.env -%}
+{%- endif -%}
+APP_ENV={{ app_env }}
```

Comment:
is this actually useful? Wasn't provided before

Reply:
Wrong, it was provided before through `fastcgi_param` and in various other places.